README.md

```diff
@@ -22,26 +22,28 @@ More information:
 
 ## Contents
 
-* [What's new in Zowe Explorer 1.6.0](#what's-new-in-zowe-explorer-1.6.0)
+* [What's new in Zowe Explorer 1.7.0](#what's-new-in-zowe-explorer-1.7.0)
 * [Prerequisites](#prerequisites)
 * [Getting Started](#getting-started)
 * [Credentials Security](#credentials-security)
 * [Usage Tips](#usage-tips)
 * [Extending Zowe Explorer](#extending-zowe-explorer)
 
-## What's new in Zowe Explorer 1.6.0
+## What's new in Zowe Explorer 1.7.0
 
 New features:
 
-* Create profiles with different, other than `zosmf`, profile types. For example, if you have the FTP plug-in installed, you can create a profile of the FTP type.
-* Improved user experience around adding new profiles. If the quick pick list does not contain any profiles, you will be prompted to create a new profile instead of picking the option to create a new profile from the empty list.
+* Added the credentials check feature that allows update credentials if a user gets an authorization error.
+* Added the star icon button that allows add data sets, USS files, or jobs to the corresponding list of favorites
```

Comment:
This is confusing. 

Added a star icon which clearly denotes data sets, USS files, and jobs as favorites.